### PR TITLE
Fixes for deployment procedure

### PR DESCRIPTION
**Pull request by Marcin-Radecki**
* Backing up tokens on AZERO side for `bAZERO` in both prod and test versions
* by default, all configs are prod ones
* copy-paste of `addTokenPair`

### CODE REVIEW

File: azero/scripts/2_l2_setup.ts [hunk 1]
```diff
@@ -40,20 +40,20 @@ async function main(): Promise<void> {
   console.log(`wAZERO address set to ${wAzeroAddress}`);
   await most.tx.setWazero(wAzeroAddress);
 
-  if (config.dev) {
-    console.log("Creating WrappedAzero instance");
-    const wAzero = new WrappedAzero(wAzeroAddress, deployer, api);
-    const deposit = BigInt(10000 * 10 ** 12);
+  console.log("Creating WrappedAzero instance");
+  const wAzero = new WrappedAzero(wAzeroAddress, deployer, api);
+  const deposit = BigInt(10000 * 10 ** 12);
```
Comment (kostekIV) [inline azero/scripts/2_l2_setup.ts:45]: mv 10k to config

**Review by kostekIV — APPROVED** [review]

File: eth/hardhat.config.js [hunk 2]
```diff
@@ -163,14 +164,15 @@ if (EVM_TESTNET_KEY) {
...
-      bazero_token_mint: 10e6, // Mint 1 million tokens - will be scaled by decimals.
+      bazero_token_mint: 10_000,
       pool_config: {
         amplification_coefficient: 2000,
         admin_fee: 0,
         fee: 0,
         initial_token_supply: 10_000,
         initial_native_supply: 10_000,
       },
+      wazero_l1_address: "5EFDb7mKbougLtr5dnwd5KDfZ3wK55JPGPLiryKq4uRMPR46"
```
Comment (kostekIV) [inline eth/hardhat.config.js:175]:
```suggestion
      wazero_l1_address: "5EFDb7mKbougLtr5dnwd5KDfZ3wK55JPGPLiryKq4uRMPR46",
```

Comment (Marcin-Radecki) [reply eth/hardhat.config.js]: Linter caught this as well. Fixed

File: azero/scripts/2_l2_setup.ts (hunk 1 above)
Comment (Marcin-Radecki) [reply azero/scripts/2_l2_setup.ts]: Done. I also added example `l2_testnet.json.example` file

**Review by kostekIV — APPROVED** [review]

**Pull request merged**